src/components/TitleBar/ViewTitleBar.js

```diff
@@ -22,40 +21,10 @@ import {
     sGetDashboardById,
     sGetDashboardItems,
 } from '../../reducers/dashboards';
-import { colors } from '@dhis2/ui-core';
 
-const NO_DESCRIPTION = i18n.t('No description');
+import classes from './styles/ViewTitleBar.module.css';
 
-const styles = {
-    actions: {
-        display: 'flex',
-        alignItems: 'center',
-        marginLeft: '20px',
-    },
-    starIcon: {
-        fill: colors.grey600,
-    },
-    textButton: {
-        minWidth: '30px',
-        top: '1px',
-    },
-    editLink: {
-        display: 'inline-block',
-        verticalAlign: 'top',
-        textDecoration: 'none',
-        marginRight: '4px',
-    },
-    titleBar: {
-        display: 'flex',
-        alignItems: 'flex-start',
-    },
-    titleBarIcon: {
-        marginLeft: 5,
-        position: 'relative',
-        top: 1,
-        cursor: 'pointer',
-    },
-};
+const NO_DESCRIPTION = i18n.t('No description');
```

Comment:
Seems like this is only used once, should it still be a const?

Reply:
Good point